### DISCUSSION TITLE
Setup once installed - sorry for my ignorance

### DISCUSSION THREAD

**Discussion by J4YRD** [FaserF/hassio-addons#264] [Q&A]
Hey FaserfF, 

Sorry, but I have been trying to get right all weekend, but I just cant get it. 

I have installed the differnet versions of your apache2 add-on, but can get the demo page to come up. I went into the config page and  tried a heap of different combinations. I am trying to use this for the https://github.com/viny182/KFloorP Home Assistant Kindle Dashboard, but am really stuck. 

I usually work this kind of thing out, but have had no luck. 

What is the simplest combinations of setting to get the demo page up? I should be able to work it out from there. 

I get: 
502: Bad Gateway
Apache2 Minimal
The add-on seems to not be ready, it might still be starting. Do you want to try again?

If you have a set of instructions for absolute novices, it would be great.

**Comment by FaserF** [comment]
Hey,
had you tried accessing your webpage only from ingress or can you also try accessing it with your link f.e. http://your_local_ip_adress_of_ha_server

If the webpage opens there fine, then your page is not ingress compatible, see: https://github.com/FaserF/hassio-addons/tree/master/apache2#ingress

If the webpage is not accessable from there too, then something seems to be wrong with the installation of the web files.
